_includes/contribute-story.html

```diff
@@ -11,5 +11,52 @@
       Contribute a Story
     </a>
   </p>
-  <img class="u-posAbsoluteBottomRight u-size2of5 u-sm-size1of3 u-sm-spaceRight1 u-md-size1of4" src="https://cloudfour.com/wp-content/themes/cloudfour/static/toolkit/images/portland.svg" alt="Portland, Oregon landscape">
+
+  <!--
+    This SVG matches the Portland Landscape image in the cloudfour.com footer,
+    but uses an inline SVG instead of an <img> due to this project's different
+    structure.
+
+    It is purely decorative, so has `aria-hidden="true"` (matching the empty alt
+    tag on cloudfour.com)
+
+    It has a width attribute, so if CSS doesn't load it will display at a
+    reasonable size. It does _not_ have a height attribute. Since our CSS isn't
+    modifying the height, a height attribute would remain in effect, shrinking
+    the SVG on large screens.
+  -->
```

Comment:
This seems like a pretty verbose comment to output to the site? 🤔

If the inline SVG requires this much more explanation, I'm not sure it's preferable to the data URI?

Reply:
That's fair. I guess I'm spoiled by templating engines and build tools that strip out comments like this 😅 

I'm not sure the comment is necessary. I'll take a look and see if there are parts that should be kept and shortened, or if the whole thing should be yanked out.

I don't have a strong preference between the inline SVG and the data URI.

Reply:
Good call @tylersticka . Re-reading it, most of that comment was unnecessary. I stripped out most of it but left a couple lines explaining what the graphic was so devs don't need to parse the SVG to understand it. I'm happy to make further changes or remove if completely if you'd prefer. 

Mind taking another look? Thanks 🙂

Reply:
@Paul-Hebert I'm not super familiar with Jekyll, but I think it uses Liquid? Did you try wrapping the comment in [Liquid comment tags](https://shopify.github.io/liquid/tags/comment/)?

Reply:
Oh good catch. Thanks. I was thrown off by the `.html` file ending. Lemme see...

Reply:
Just switched it to a liquid comment 🙂